tests/sub_group_mask/sub_group_mask_common.h

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Extension Conformance Test
+//
+//  Provides common methods for sub_group_mask tests
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_SUB_GROUP_MASK_COMMON_H
+#define __SYCLCTS_TESTS_SUB_GROUP_MASK_COMMON_H
+
+#include "../common/common.h"
+#include "../common/type_coverage.h"
+
+#ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
+namespace {
+
+static const auto types =
+    named_type_pack<char, signed char, unsigned char, short, unsigned short,
+                    int, unsigned int, long, unsigned long, long long,
+                    unsigned long long>{
+        "char",           "signed char", "unsigned char",     "short",
+        "unsigned short", "int",         "unsigned int",      "long",
+        "unsigned long",  "long long",   "unsigned long long"};
+
+template <typename funT, typename PredT, typename T>
+class test_kernel;
+
+constexpr size_t globalSize = 128;
+constexpr size_t localSize = 32;
```

Comment:
```suggestion
constexpr size_t globalSize = 8;
constexpr size_t localSize = 4;
```
Why these values? Shouldn't they depend on sub-group size supported by HW?

Reply:
I've applied interactions with different `sub_group_size` in this commit: eb0ad1d745453a6be2666cc4e162a173debfb675
If this fix is OK, then I'll apply this change to another PRs

Reply:
@bader this fix (commit: eb0ad1d745453a6be2666cc4e162a173debfb675) is ok? can I apply this change to another tests on `sub_group_mask`?

Reply:
@vasilytric, it doesn't really answers my question. I see that now test tries sub-group sizes 8, 16 and 32, so having I think I get why we set `localSize` to 32, but it's not clear why global size should be larger than the `localSize`.

Reply:
Yes, you are right we can have `global_size` the same as `local_size` and in this case we are going to have fewer parallel processes
This fix could be found in this commit: 96661372cff541955c3fa44fdaa97b535636de97